load/PreValidate.py

```diff
@@ -427,59 +199,48 @@ def hasErrors(self):
 ## aws s3 ls --profile DBP_DEV s3://dbp-etl-mass-batch
 ## 328 stock numbers, 9094 files
 
-## System Test for PreValidate
+## System Test for PreValidate,testing the validateDBPETL entry point (see PreValidateLambdaHandlerStub for the lambda entry point system test)
 if (__name__ == "__main__"):
 	import boto3
 	from Config import *
 	from AWSSession import *
 
-	if len(sys.argv) < 3:
-		print("FATAL command line parameters: environment from_text_processing_folder ")
+	if len(sys.argv) < 2:
+		print("FATAL command line parameters: environment location prefix directoryName")
 		sys.exit()
 
-	prefix = sys.argv[2][:-1] if sys.argv[2].endswith("/") else sys.argv[2]
+	location = "s3://dbp-etl-upload-dev-ya1mbvdty8tlq15r"
+	prefix = "2022-03-25-16-14-34"
+	directoryName = "Abidji_N2ABIWBT_USX"
+
+	if len(sys.argv) > 2:
+		location = sys.argv[2][:-1] if sys.argv[2].endswith("/") else sys.argv[2]
+	
+	if len(sys.argv) > 3:
+		prefix = sys.argv[3]
+
+	if len(sys.argv) > 4:
+		directoryName = sys.argv[4]
+
+	print ("location [%s], prefix [%s], directoryName [%s]" % (location, prefix, directoryName))
+	fullPath = prefix + "/" + directoryName
+
 	config = Config.shared()
 	lptsReader = LPTSExtractReader(config.filename_lpts_xml)
-	unicodeScript = UnicodeScript()
-	bucket = "etl-development-input"
 	session = boto3.Session(profile_name = config.s3_aws_profile)
 	s3Client = session.client('s3')
```

Comment:
@bradflood, Just a comment, I have done the next change about the s3Client parameter to make that the local tests work in my local.

```python
config = Config()
AWSSession.shared()
lptsReader = LPTSExtractReader(config.filename_lpts_xml)
s3Client = AWSSession.shared().s3Client
```